h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
   max-width: 500px;
+  flex-shrink: 0;
   padding-left: 0;
   padding-right: 0;
 }
 
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
```

Comment:
I would suggest using `word-break: break-all` in addition to break-word. It will not try to calculate good endings and simple roll the text over to the next row, middle of word or not. Doing that has the added benefit of consistency through the stats per section and it won't have the awkward short row for a url like `alsdfjalskdfjlaskdjflaskdfjalskdfj-b-laksdfjlaksjdfoaisdjflaksdjf` where b would be left by itself. That said, I know it applies to things like author names as well. Of which, I imagine is far less likely to be long compared to URIs. I'll leave that up to you